### PR TITLE
Miscellaneous cleanups

### PR DESCRIPTION
**Pull request by johnmccutchan**
- [x] Return the reload message even when paused at a breakpoint.
- [x] Only initialize context members if they aren't already initialized.

### CODE REVIEW

**Comment by johnmccutchan** [comment]
@tvolkert

File: packages/flutter_tools/lib/executable.dart [hunk 1]
```diff
@@ -82,10 +82,18 @@ Future<Null> main(List<String> args) async {
 
   return Chain.capture/*<Future<Null>>*/(() async {
     // Initialize globals.
-    context[Logger] = new StdoutLogger();
-    context[DeviceManager] = new DeviceManager();
-    context[DevFSConfig] = new DevFSConfig();
-    Doctor.initGlobal();
+    if (context[Logger] == null) {
+      context[Logger] = new StdoutLogger();
```
Comment (Hixie) [inline packages/flutter_tools/lib/executable.dart]: nit: no braces on one-line if statement bodies please

**Comment by tvolkert** [comment]
LGTM

File: packages/flutter_tools/lib/executable.dart (hunk 1 above)
Comment (johnmccutchan) [inline packages/flutter_tools/lib/executable.dart]: Done.

File: packages/flutter_tools/lib/executable.dart [hunk 2]
```diff
@@ -82,10 +82,18 @@ Future<Null> main(List<String> args) async {
 
   return Chain.capture/*<Future<Null>>*/(() async {
     // Initialize globals.
-    context[Logger] = new StdoutLogger();
-    context[DeviceManager] = new DeviceManager();
-    context[DevFSConfig] = new DevFSConfig();
-    Doctor.initGlobal();
+    if (context[Logger] == null) {
```
Comment (Hixie) [inline packages/flutter_tools/lib/executable.dart]: if context is a Map, consider putIfAbsent or ??=

**Pull request merged**